app/femr/common/models/PatientEncounterItem.java

```diff
@@ -37,9 +37,14 @@
 
     private String medicalDateOfVisit;
     private String physicianEmailAddress;
+    private String physicianFullName; //Andrew Change
 
     private String pharmacyDateOfVisit;
     private String pharmacistEmailAddress;
+    private String pharmacistFullName; //Andrew Change
+
+    private String nurseFullName; //Andrew Change
+
```

Comment:
Just a thought: Would it be better to use UserItem here for physician, pharmacist, and nurse?

if we do 

private UserItem physician;
private UserItem pharmacist;
private UserItem nurse;

We'll have access to all the user info, which might be helpful. This definitely isn't necessary and what is here works just fine.